configure.ac

```diff
@@ -1415,7 +1415,21 @@ if test "$use_boost" = "yes"; then
 fi
 
 if test "$use_external_signer" != "no"; then
-  AC_DEFINE([ENABLE_EXTERNAL_SIGNER], [], [Define if external signer support is enabled])
+  case $host in
+    *mingw*)
+      dnl Boost Process uses Boost Filesystem when targeting Windows. Also,
+      dnl since Boost 1.71.0, Process does not work with mingw-w64 without
+      dnl workarounds. See 67669ab425b52a2b6be3d2f3b3b7e3939b676a2c.
+      use_external_signer=no
+    ;;
+    *openbsd*)
+      dnl Boost Process doesn't support OpenBSD
+      use_external_signer=no
```

Comment:
This will silently disable it even if the user specifies `--enable-external-signer`. Should error instead.

Reply:
That would make more sense to me as well, assuming I understand correctly that Boost Process will simply be absent on OpenBSD - rather than silently broken.

Reply:
>  assuming I understand correctly that Boost Process will simply be absent on OpenBSD

No, Boost Process is present on OpenBSD, but is broken.

Reply:
`build-openbsd.md` says that Boost Process is not absent, but will cause the build to fail. In any case an error makes more sense, or setting the default to false on OpenBSD.